### PR TITLE
Refactor machine credential dependent client codes

### PR DESCRIPTION
**Pull request by szkl**

### CODE REVIEW

File: AppController.coffee [hunk 1]
```diff
@@ -245,12 +245,11 @@ class IDEAppController extends AppController
...
     panel        = @workspace.getView()
     filesPane    = panel.getPaneByName 'filesPane'
-    {credential} = machineData.jMachine
 
-    if credential is KD.nick()
+    if machineData.isMine()
       rootPath   = @workspaceData?.rootPath or null
     else
-      rootPath   = "/home/#{credential}"
+      rootPath   = "/home/#{machineData.getOwner()}"
```
Comment (gokmen) [inline AppController.coffee]: It would be better to have a control here, like if there is no owner use `/` as root path.

**Comment by szkl** [comment]
Updated.

File: AppController.coffee [hunk 2]
```diff
@@ -245,12 +245,11 @@ class IDEAppController extends AppController
...
     panel        = @workspace.getView()
     filesPane    = panel.getPaneByName 'filesPane'
-    {credential} = machineData.jMachine
 
-    if credential is KD.nick()
+    if machineData.isMine()
       rootPath   = @workspaceData?.rootPath or null
     else
-      rootPath   = "/home/#{credential}"
+      rootPath   = "/home/#{machineData.getOwner() or '/'}"
```
Comment (gokmen) [inline AppController.coffee]: It will be `/home//` this time :)

Comment (szkl) [inline AppController.coffee]: Morning :facepunch:

**Comment by szkl** [comment]
Updated 2.

**Pull request merged**